concordia/static/scss/action-app.scss

```diff
@@ -5,11 +5,19 @@
 @import '../../../node_modules/bootstrap/scss/reboot';
 @import '../../../node_modules/bootstrap/scss/buttons';
 @import '../../../node_modules/bootstrap/scss/button-group';
+@import '../../../node_modules/bootstrap/scss/utilities/sizing';
 
 .btn {
     font-weight: bold !important;
 }
 
+kbd {
+    font-weight: bold;
+    border: 1px solid $gray-400;
+    background-color: $gray-200;
+    color: $gray-900;
```

Comment:
Should we be changing the `$kbd-bg` and `$kbd-color` variables instead? They're `!default` so they should be easy to override